modules/oe_theme_content_event/templates/oe-theme-content-event-online-description.html.twig

```diff
@@ -5,7 +5,9 @@
  */
 #}
 {% if description is not empty %}
-  <h3 class="ecl-u-type-heading-3">{{ 'Livestream'|t }}</h3>
-  {{ description }}
-  <a href="{{ url }}" class="ecl-u-mt-l ecl-u-mb-l ecl-link ecl-link--cta ecl-u-d-inline-block">{{ label }}</a>
+  <div {% if hidden %}class="hidden"{% endif %} data-livestream-element="true">
```

Comment:
Is `.hidden` from D core? Why do we need custom `.ecl-link--hidden` ?

Reply:
you can check this in the browser. CSS file where declared hidden class/attribute had lower priority in comparison to ECL classes for link components.